csunplugged/templates/base.html

```diff
@@ -1,161 +1,176 @@
+{% load i18n %}
 {% load static %}
 {% load render_html_field %}
+{% load django_bootstrap_breadcrumbs %}
```

Comment:
This tag is still used to generate the breadcrumbs, however we create each set separately on each page rather than use inheritance.